app/internal/adapters/routes/users.routes.js

```diff
@@ -0,0 +1,16 @@
+import express from "express";
+import { UsersController } from "../../core/controllers/users.controller";
```

Comment:
You already imported the UsersController in the controller/index.js
Use this path to import